docs/examples/tls_mutual_authentication.rst

```diff
@@ -17,7 +17,8 @@ tls_example.py::
     context.load_cert_chain("PIKA_DIR/testdata/certs/client_certificate.pem",
```

Comment:
Why not change all arguments to named arguments?

Reply:
All the arguments on the changed call have names.

Reply:
I mean in all method calls in the example, for instance:

* `load_cert_chain`
* `SSLOptions`